README.md

```diff
@@ -27,7 +27,7 @@ particular, you can provide a custom:
 
 ## Requirements
 
-FastBoot App Server requires Node.js v4.2 or later.
+FastBoot App Server requires Node.js v8 or later.
```

Comment:
Haha, I'd like to bump this again too but seems good to do separately...

Reply:
Yes 😬, #123 is the next step.